app/helpers/riiif_helper.rb

```diff
@@ -31,6 +31,52 @@ def riiif_image_srcset_pixel_density(riiif_file_id, base_width, format: 'jpg', q
     end.join(", ")
   end
 
+  # create an image tag for a 'member' (could be fileset or child work) thumb,
+  # for use on show page. Calculates proper image tag based on lazy or not,
+  # use of riiif for images or not, and desired size. Includes proper
+  # attributes for triggering viewer, analytics, etc.
+  #
+  # if use_image_server is false, size_key is ignored and no srcsets are generated,
+  # we just use the stock hydra-derivative created image labelled 'jpeg'
+  def member_image_tag(member, size_key: nil, lazy: false, use_image_server: CHF::Env.lookup(:use_image_server_on_show_page))
+    base_width = size_key == :large ? 525 : 208
+
+    args = {
+      class: ["show-page-image-image"],
+      alt: "",
+      data: {
+        trigger: "chf_image_viewer",
+        member_id: member.id,
```

Comment:
uh... i'm not sure which is right. I think thsi is right now and the other one was wrong, though. Should have collapsed those branches before, too many opportunities for inconsistency and bugs!